src/features/authorSlice.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Avoid circular dependencies. Instead of disabling eslint rule, refactor your code to eliminate the circular dependencies.